### ISSUE TITLE
Implementation of Default trait for records with Uuid fields is invalid

### ISSUE THREAD

**Issue by gfmedeiros** [lerouxrgd/rsgen-avro#22]
Given the schema:

```json
{
  "type": "record",
  "name": "ExampleRecord",
  "fields": [
    {
      "name": "foo",
      "type": "string",
      "logicalType": "uuid"
    }
  ]
}
```

The following code is generated:

```rust
#[derive(Debug, PartialEq, Clone, serde::Deserialize, serde::Serialize)]
#[serde(default)]
pub struct ExampleRecord {
    pub foo: uuid::Uuid,
}

impl Default for ExampleRecord {
    fn default() -> ExampleRecord {
        ExampleRecord {
            foo: 00000000-0000-0000-0000-000000000000,
        }
    }
}
```

This fails to compile because of `00000000-0000-0000-0000-000000000000` on line 11, rustc thinks it's a sequence of integers:
  
> expected struct \`Uuid\`, found integer

I appreciate your work. Let me know if I can be of any help.

**Comment by lerouxrgd** [comment]
Thanks for reporting this.

It should be fixed in `0.9.8`.

**Comment by lerouxrgd** [comment]
Closing as it should be fine.
